src/bun.js/bindings/bindings.cpp

```diff
@@ -3715,13 +3715,20 @@ JSC__JSValue JSC__JSValue__getIfPropertyExistsImpl(JSC__JSValue JSValue0,
 {
 
     JSValue value = JSC::JSValue::decode(JSValue0);
-    if (UNLIKELY(!value.isObject()))
+    if (UNLIKELY(!value)) {
```

Comment:
I think this should be a debug assertion instead of an extra branch. We control usage of this function, it should never receive zero